front/src/modules/pathfinding/utils.ts

```diff
@@ -36,6 +39,31 @@ export const formatSuggestedOperationalPoints = (
     coordinates: getPointCoordinates(geometry, pathLength, op.position),
   }));
 
+export const fetchTrackNamesForSuggestedOperationalPoints = async (
```

Comment:
Nit : for me, it implies that the function returns only the track names. Maybe we could try this (only suggestion)

```suggestion
export const addTrackNamesToSuggestedOperationalPoints = async (
```

Reply:
Sure! By calling it fetch I wanted to emphasize the api call, but the fact the function is async should be enough clues.